BraintreeCore/src/main/java/com/braintreepayments/api/core/DeviceInspector.kt

```diff
@@ -63,7 +63,7 @@ class DeviceInspector(
         return appHelper.isAppInstalled(context, PAYPAL_APP_PACKAGE)
     }
 
-    fun isVenmoInstalled(context: Context?): Boolean {
+    fun isVenmoInstalled(context: Context): Boolean {
```

Comment:
This is probably a super basic question but is there a reason Venmo requires a context and PayPal does not for the app install checks?